src/components/Pricing/ButtonContent.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+export function ButtonContent({ loadingStripeCheckout, currentPlan, id }) {
+  if (loadingStripeCheckout === id) {
+    return (
+      <div
+        className="animate-spin inline-block w-5 h-5 border-[3px] border-current border-t-transparent rounded-full text-blue-600 "
+        role="status"
+        aria-label="loading"
+      >
+        <span className="sr-only">Loading...</span>
```

Comment:
@paulsp94 FYI: In legaliser, we are using `Loading...` text instead of spinner. I'll add a spinner in VBS at this point.